src/app/backend/replicasetdetail.go

```diff
@@ -157,6 +163,24 @@ func DeleteReplicaSetWithPods(client *client.Client, namespace string, name stri
 	return nil
 }
 
+// Updates number of replicas in Replica Set based on Replica Set Spec
+func UpdateReplicasCount(client *client.Client, namespace string, name string,
```

Comment:
Ok, so i think other functions in this file should also use this interface to be testable. Update this in this PR?